aws-datastore/src/main/java/com/amplifyframework/datastore/AWSDataStorePlugin.java

```diff
@@ -132,18 +129,10 @@ public String getPluginKey() {
     public void configure(
             @Nullable JSONObject pluginConfiguration,
             @NonNull Context context
-    ) throws DataStoreException {
-        try {
-            this.pluginConfiguration =
-                AWSDataStorePluginConfiguration.fromJson(pluginConfiguration);
-        } catch (DataStoreException badConfigException) {
-            throw new DataStoreException(
-                "There was an issue configuring the plugin from the amplifyconfiguration.json",
-                badConfigException,
-                "Check the attached exception for more details and " +
-                    "be sure you are only calling Amplify.configure once"
-            );
-        }
+    ) {
+        // AWSDataStorePlugin does not consider any values from amplifyconfiguration.json.
+        // To effect the behavior of the plugin, provide an {@link DataStoreConfiguration}
```

Comment:
Hmm i'm pretty sure we should: s/effect/affect

Also, this is a bit surprising to me... but, have we decided we are not going to accept values from amplifyconfiguration.json ?